src/java/htsjdk/variant/vcf/VCFHeaderLineTranslator.java

```diff
@@ -83,13 +83,29 @@
         // are we inside a quotation? we don't special case ',' then
         boolean inQuote = false;
 
+        // if we are in a quote and we see a backslash followed by quote, treat it as an escaped quote
+        boolean escape = false;
+
         // a little switch machine to parse out the tags. Regex ended up being really complicated and ugly [yes, but this machine is getting ugly now... MAD]
         for (char c: valueLine.toCharArray()) {
-            if ( c == '\"' ) {
-                inQuote = ! inQuote;
+            if ( c == '\"') {
+                if (escape) {
+                    builder.append(c);
+                    escape = false;
+                } else {
+                    inQuote = !inQuote;
+                }
             } else if ( inQuote ) {
-                builder.append(c);
+                if (escape) {
+                    builder.append(c);
```

Comment:
I don't see anything in the spec that says what to do with a backslash preceeding a character other than double quote or backslash, so I've made the parser code not do anything special in that case.